code/index.html

```diff
@@ -1,21 +1,173 @@
 <!DOCTYPE html>
-<html>
-
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
-  </head>
-  <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
-
-      <img src="example-recipe-background.png" alt="Banana Bread">
-
-    -->
-    <h1>Hello there! 👋🏼</h1>
-  </body>
-</html>
\ No newline at end of file
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Tamira Leber - Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <a id="top"></a>
+    <div class="navigationLink">
+        <div class="navElement">
+            <a id="nav1" class="navigationLink" href="./index.html">Home</a>
+        </div>
+        <div class="navElement">
+            <a id="nav2" class="navigationLink" href="./projects.html">Projekte</a>
+        </div>
+        <div class="navElement">
+            <a id="nav3" class="navigationLink" href="./contactme.html">Contact</a>
+        </div>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Rozha+One&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
+    </div>
+</head>
+<body>
+
+<div class="intro">
+    <div class="welcome">
+        <h1>Tamira Leber</h1>
+        <h2>
+            <div>Digital Product Management Assistant</div>
+            <div>Studentin BSc Informatik - iCompetence</div>
+        </h2>
```

Comment:
Instead of having divs inside the h2, you could achieve the same result by having two h2s